src/bankid.ts

```diff
@@ -131,7 +136,7 @@ export enum BankIdMethod {
 }
 
 export type BankIdRequest =
```

Comment:
This will make this type makes it a bit hard to use for V6 🤔 
Wonder if we should just do a breaking change and remove this type in favor of exposing the individual types

Reply:
The way the code is right now, `BankIdRequest` is only used in a private `#call` method, so exporting it makes no sense to me, although a union of both v5 and v6 types would (for the time being), be correct 😬 . 

For the purpose of migrating from v5 to v6 it would be nice to export both clients in a  minor release for a brief moment, then make a "major" release where we get the chance to implement some learnings or feedback from others with regards to naming, and remove the v5 client for real.

For example the types of `collect` could be gathered through `Parameters<>` , so maybe we could leave that before exporting new individual types? 

e.g.
```ts
const b = new BankIdClient();
const c = new BankIdClientV6();

type AuthParams = Parameters<typeof b.authenticate>[0]
type AuthParamsV6 = Parameters<typeof c.authenticate>[0]
```